### ISSUE TITLE
fail to use micropython on esp32

### ISSUE THREAD

**Issue by yuxiangyuzm** [pycom/pycom-micropython#59]
when i flash the wipy.bin(it's a file what we can use python on esp32 ) into esp32, it is successful, but when i use it,  there are something wrong, just like this(on putty):  
                               >a=3
                               >print(a)
                               3
                               >import os
                               what?
                               >import sys
                               what?
                               >
i do not know why. 
  ![image](https://cloud.githubusercontent.com/assets/24452528/21873356/01b87886-d8a9-11e6-962e-f0959ea81b40.png)

Please include the following information when submitting a bug report:

* The Pycom board you are using (e.g. WiPy 2.0, LoPy).
* The firmware version you are using. You can get it by executing the following Python code at the REPL:
```
import os
os.uname()
```
The output will look similar to this:
```
(sysname='LoPy', nodename='LoPy', release='0.9.4.b1', version='bc42ea7 on 2016-11-14', machine='LoPy with ESP32')
```
* Exact steps to cause this issue
    1. First I did this...
    2. Then, I did that...
* What you expected
    * e.g. I expected *abc* to *xyz*
* What happened instead
    * e.g. Instead, I got *zyx*

Thank you!

**Comment by nevercast** [comment]
I thought the REPL had >>> not >

**Comment by yuxiangyuzm** [comment]
yes, but what result in this problem. the bin file?or something esle? @nevercast

**Comment by nevercast** [comment]
Try using the latest firmware. Have you used the firmware updater?

**Comment by yuxiangyuzm** [comment]
@nevercast ok i  will have a try , but two days ago ,i did this. i think it's latest firmware. thank you!

**Comment by yuxiangyuzm** [comment]
@nevercast i tried just now. there is also this problem. 
![image](https://cloud.githubusercontent.com/assets/24452528/21878028/4cd67130-d8ca-11e6-8905-99e77bf4a334.png)

**Comment by negativekelvin** [comment]
@nevercast yes that's the tinybasic prompt

Its a flash configuration issue. Suggest to build firmware from source for running on non-pycom hardware

**Comment by DirkHeinke** [comment]
Any success running pycom-micropython on a ESP32 dev board?
